### ISSUE TITLE
Doubled automation for hass-auto-backup can not be purged

### ISSUE THREAD

**Issue by michael5411** [jcwillox/hass-auto-backup#113]
### The problem

I’m using all the latest and up-to-date versions of HA, HACS and today I updated hass-auto-backup. It’s checked and OK.

BEFORE building a new backup automation in HA,  I realized, that there where TWO automations of my backup-script named:  "Backup täglich HACS“. I never built two of them.

When trying to open the first of these scripts, there came a message: „… can only be edited in YAML“. But there is no accessible YAML. At the entity-menu I get the info, that there is no unique entity-ID for it .. etc.

As a result I have an automation, that is running fine „Backup täglich HACS_neu“ and an 2nd automation, that is **NOT part of automation.YAML** and therefore, it is by no means (that are familiar to me) possible to delete it. I’ve deactivated it. 

But: HOW DO I DELETE IT?

Looks for me like the „header of the automation“ was separated from the YAML - possibly when updating hass-auto-backup - and now I’ve no access to the header and therefore I got „a dead lame duck“ of an automation header without context.

Here is the YAML of my (working) automation. It is exactly the same YAML, that was used for the no erasable automation (the alias is slightly different):

```
alias: Backup täglich HACS_neu
description: ""
trigger:
  - platform: time
    at: "01:55:00"
condition: []
action:
  - service: auto_backup.backup_full
    data:
      name: Auto Backup HACS
      keep_days: 3
mode: single
```

### What version of Auto Backup has the issue?

lastest update

### What version of Home Assistant are you running?

lastest

### What type of installation are you running?

Home Assistant Container

### If you're running HA OS/Supervised, what version of the Supervisor are you running?

lastest

### Example YAML snippet

```yml
alias: Backup täglich HACS_neu
description: ""
trigger:
  - platform: time
    at: "01:55:00"
condition: []
action:
  - service: auto_backup.backup_full
    data:
      name: Auto Backup HACS
      keep_days: 3
mode: single
```
```


### Anything in the logs that might be useful for us?

_No response_

### Additional information

_No response_

**Comment by henriklund** [comment]
As far as I understand hass-auto-backup does not generate automations for you and also does not alter existing automations (even when settings get deprecated). My gut feeling tells me this is is more along the lines of an issue with Home Assistant. From your description I reckon that you have seen both of the daily backup automation ('Backup täglich HACS') in Settings -> Automations, correct? A few things I would check (if not already done):
- is 'automation: !include automations.yaml' explicitly mentioned in your configuration.yaml file?
- do you get one or multiple hits if you run following (from Terminal) in the config folder of your installation?
`grep -r "Backup täglich HACS" *
`
If above does not work, an alternative would be 
`find . -name "*" -type f -exec grep "Backup täglich HACS" {} + 
`
If you get multiple hits, you need to check these.

**Comment by michael5411** [comment]
Thanks for your excellent analysis and your hints. Using your advice I realized, that I was again successfully cheating myself.

I’m running a second home assistant for my electronics lab. It’s called „labassistant“ and it does backups as my home assistant does to which it is connected by „Remote Home-Assistant“. I did never realize, that automations are „mirrored“ to list of automations in the „master system“. But they are not accessible from there. 

Therefore it is impossible to find the automation in „automations.yaml“ of my „home assistant“, because it is located in „labassistant“. Lots of headache - easy solution“ :-) 

Thanks for your help!
